CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ celeritas_optional_package(OpenMP "Enable CPU shared-memory parallelism")
 celeritas_optional_package(Python "Use Python to generate and preprocess")
 celeritas_optional_package(ROOT "Enable ROOT I/O")
 celeritas_optional_package(VecGeom "Use VecGeom geometry")
+celeritas_optional_package(Covfie "Use Covfie field integrator")
```

Comment:
Check lower in the file for additional places to add to: `Find${_dep}.cmake`, Covfie_DIR,